src/docs/truffle/reference/truffle-commands.md

```diff
@@ -361,6 +361,8 @@ Options:
 * `<test_file>`: Name of the test file to be run. Can include path information if the file does not exist in the current directory.
 * `--compile-all`: Compile all contracts instead of intelligently choosing which contracts need to be compiled.
 * `--compile-all-debug`: Like `--compile-all`, but compiles contracts in debug mode for extra information.  Has no effect on Solidity <0.6.3.
+* `--compile-none`: Skip all contract compilations, this option is not intended for use with most tests, which will require compilation.
```

Comment:
This sentence is a comma splice.  It should be split into two sentences.  (Or you could use a semicolon, but I think this should probably be split in two.)

Reply:
Split into two, thx